integration-cli/docker_cli_exec_test.go

```diff
@@ -539,10 +540,12 @@ func (s *DockerSuite) TestExecWindowsOpenHandles(c *check.C) {
 	testRequires(c, DaemonIsWindows)
 
 	if runtime.GOOS == "windows" {
+		// Note we parse kernel.GetKernelVersion rather than osversion.Build()
+		// as test binaries aren't manifested, so would otherwise report build 9200.
 		v, err := kernel.GetKernelVersion()
 		assert.NilError(c, err)
 		build, _ := strconv.Atoi(strings.Split(strings.SplitN(v.String(), " ", 3)[2][1:], ".")[0])
-		if build >= 17743 {
+		if build >= osversion.RS5 {
 			c.Skip("Temporarily disabled on RS5 17743+ builds due to platform bug")
```

Comment:
Changed this to `RS5+` (`17743` looks to be a pre-release)

@jhowardmsft is this platform bug still current?

Reply:
AFAIK, yes.